client/src/cb-admin/pages/Settings.js

```diff
@@ -112,15 +123,69 @@ export default class SettingsPage extends React.Component {
           this.props.history.push('/admin/login');
         }
       });
+
+    Visitors.get(this.props.auth)
+      .then((res) => {
+        this.props.updateAdminToken(res.headers.authorization);
```

Comment:
Is it necessary to update the token after _every_ api call? Does the server change the token?